test/end-to-end.js

```diff
@@ -982,55 +982,55 @@ describe('end-to-end', function() {
     }
   });
 
-  it('includes sensitive error details in development mode', function(done) {
-    var loopback = require(SANDBOX + '/node_modules/loopback');
-    var boot = require(SANDBOX + '/node_modules/loopback-boot');
-    var app = loopback({ localRegistry: true, loadBuiltinModels: true });
-    var bootOptions = {
-      appRootDir: SANDBOX + '/server',
-      env: 'development'
-    };
-    boot(app, bootOptions, function(err) {
-      if (err) return done(err);
-      request(app)
-        .get('/url-does-not-exist')
-        .expect(404)
-        .end(function(err, res) {
-          if (err) return done (err);
-          var responseBody = JSON.stringify(res.body);
-          expect(responseBody).to.include('stack');
-
-          done();
-      });
-    });
-  });
-
-  it('omits sensitive error details in production mode', function(done) {
-    var loopback = require(SANDBOX + '/node_modules/loopback');
-    var boot = require(SANDBOX + '/node_modules/loopback-boot');
-    var app = loopback({ localRegistry: true, loadBuiltinModels: true });
-    var bootOptions = {
-      appRootDir: SANDBOX + '/server',
-      env: 'production'
-    };
-    boot(app, bootOptions, function(err) {
-      if (err) return done(err);
-      request(app)
-        .get('/url-does-not-exist')
-        .expect(404)
-        .end(function(err, res) {
-          // Assert that the response body does not contain stack trace.
-          // We want the assertion to be robust and keep working even
-          // if the property name storing stack trace changes in the future,
-          // therefore we test full response body.
-          if (err) return done(err);
-          var responseBody = JSON.stringify(res.body);
-          expect(responseBody).to.not.include('stack');
-
-          done();
-        });
-    });
-  });
+  // it('includes sensitive error details in development mode', function(done) {
```

Comment:
@loay: Instead of commenting out the entire test case, it would be great to use `it.skip` or `describe.skip`.